src/components/Textarea/Textarea.tsx

```diff
@@ -1,6 +1,6 @@
 import classNames from "classnames";
 import React, { useEffect, useRef } from "react";
-import type { HTMLAttributes, ReactNode } from "react";
+import type { InputHTMLAttributes, ReactNode } from "react";
```

Comment:
Looks like there's also `TextareaHTMLAttributes` which seems like it would make more sense?

Reply:
Oh nice thanks, I even looked!